src/components/RangeFilm/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Range } from 'rc-slider'
+import { FILM_ADVERTISING } from '../../constans'
+
+const { MIN } = FILM_ADVERTISING
+
+const RangeFilm = ({
+  field,
+  setValueFilm,
+  value,
+  max,
+  marksMin,
+  marksMax,
+  marksMinLabel,
+  marksMaxLabel,
+}) => {
+  return (
+    <Range
+      {...field}
+      onChange={(value) => {
+        setValueFilm(value)
+        field.onChange(value)
+      }}
```

Comment:
it is good practice to have clean JSX, without pure expressions/variable declarations/etc in props
since each rerender this expressions will be recalculated, you have to use `useMemo` and `useCallback`

```
const handleChange = useCallback((value) => {
        setValueFilm(value)
        field.onChange(value)
}, [])

...

onChange={handleChange}
```
`eslint` will tell you about `deps` (`[]` - second argument in`useCallback` and `useMemo`)
you have to aways add those dependencies

https://ru.reactjs.org/docs/hooks-reference.html#usecallback
https://ru.reactjs.org/docs/hooks-reference.html#usememo

read more about those hooks, to know when you should use `useCallback` and when `useMemo`